src/Microsoft.ML.Transforms/Text/WordEmbeddingsTransform.cs

```diff
@@ -21,10 +23,16 @@
 [assembly: LoadableClass(typeof(WordEmbeddingsTransform), null, typeof(SignatureLoadDataTransform),
```

Comment:
>typeof(WordEmbeddingsTransform) [](start = 25, length = 31)

should be typeof(IDataView), typeof(WordEmbeddingTransform)

Reply:
not just IDataVIew, but also WordEmbeddingTransform

---
In reply to: [217929112](https://github.com/dotnet/machinelearning/pull/928#discussion_r217929112) [](ancestors = 217929112)

Reply:
Sorry for all this confusion, but I was confused as well. First type should be return type of your static Create method. And since you use IDataTransform as return type, you need to use IDataTransform

---
In reply to: [218249508](https://github.com/dotnet/machinelearning/pull/928#discussion_r218249508) [](ancestors = 218249508,217929112)